tsdef/twilio-video-tests.ts

```diff
@@ -230,6 +230,15 @@ let room: Video.Room | null = null;
 let localVideoTrack: Video.LocalVideoTrack | null = null;
 let localAudioTrack: Video.LocalAudioTrack | null = null;
 
+// Testing finally method
+Video.connect('$TOKEN', {
+  name: 'room-name',
+  video: false,
+  audio: false
+}).finally(() => {
```

Comment:
Do we have tests for `catch` and `then` in this file? It might be good to chain them here.